packages/smart-vaults/src/vault/SmartVault.sol

```diff
@@ -35,13 +35,35 @@ contract SmartVault is IAccount, Ownable, UUPSUpgradeable, MultiSignerAuth, ERC1
         ERC1271
     }
 
+    /// @notice Upper limits for maxPriorityFeePerGas, preVerificationGas, verificationGasLimit, callGasLimit,
+    /// paymasterValidationGasLimit and paymasterPostOpGasLimit that should be charged by the userOp. This is included
+    /// in the light userOp hash to ensure last signer does not exceed the specified gas price/limits. These values will
+    /// be ignored when threshold is 1. paymaster, paymasterValidationGasLimit and paymasterPostOpGasLimit will be
+    /// ignored if paymasterAndData is empty.
+    struct LightUserOpGasLimits {
+        uint256 maxPriorityFeePerGas;
+        uint256 preVerificationGas;
+        uint256 callGasLimit;
+        uint256 verificationGasLimit;
+        address paymaster;
+        uint256 paymasterValidationGasLimit;
```

Comment:
In the EIP this is called `paymasterVerificationGasLimit`. I was thrown off by the different nomenclature so I would consider switching to the standard term.

The reference implementation uses "validation" in some parts but this must be left over from some previous version: https://github.com/eth-infinitism/account-abstraction/blob/048d8b0fc9eb681f8a267a8de55bfb45ea4cfc2d/contracts/core/UserOperationLib.sol#L110-L113